### ISSUE TITLE
Setting UITextField Height smaller than 32 pixels

### ISSUE THREAD

**Issue by edvv** [Microsoft/WinObjC#2802]
The UITextField XAML backing ( _textField) can not draw a height less than 32 pixels. If the height is less than 32 then its box is clipped on the bottom. Is there a way to get it to draw to a smaller height, such as 21 or 24 pixels?

Sincerely,

Ed

**Comment by rajsesh-msft** [comment]
@edvv Are you setting the minimumFontSize on the UITextField?

**Comment by edvv** [comment]
@rajsesh-msft No. Attached is a project and two images of that project's app window on Windows and the iPhone showing the issue.

Sincerely,

Ed

iPhone:

![iphone](https://user-images.githubusercontent.com/15889800/31817395-beae9a52-b561-11e7-91ce-6218ebc58485.jpg)

Windows:

![windows](https://user-images.githubusercontent.com/15889800/31817491-30cab076-b562-11e7-8bf3-ea1fd2051498.jpg)


Project:

[textfield.zip](https://github.com/Microsoft/WinObjC/files/1401634/textfield.zip)

**Comment by edvv** [comment]
@rajsesh Do you have any idea on how to work around this bug?
